vispy/visuals/volume.py

```diff
@@ -929,7 +929,7 @@ def clipping_planes(self, value):
             value = np.empty([0, 2, 3])
         self._clipping_planes = value
 
-        self._clip_func = self._build_clipping_planes_func(len(value))
+        self._clip_func = Function(self._build_clipping_planes_func(len(value)))
```

Comment:
Can we change the name from `_func` to something else? Maybe `_glsl`?

Reply:
Done!